perfil/core/models.py

```diff
@@ -244,12 +244,9 @@ def rosies_suspicions(self):
         return self.politician.rosies_suspicions if self.politician else []
 
     def image(self):
-        if self.year != 2018:
-            return None
-
```

Comment:
Verifiquei aqui que fotos só começaram a ser disponibilizadas a partir de 2014. Na nossa base teremos apenas 2018 e 2022 por enquanto. Então sugiro voltar com a limitação hardcoded por enquanto com `if self.year not in [2018, 2022]`.

Reply:
Futura melhoria seria verificar se o arquivo existe no DO e colocar None caso contrário.

Reply:
Os anos hard-coded também não resolvem necessariamente. Não acho que podemos garantir que o TSE irá disponibilizar as fotos de todos os candidatos, sendo assim essa é uma verificação que sempre terá que ser feita no front-end.
Não me parece factível fazermos uma request para checar se existe imagem ou não por conta do List Candidate, que podem ser centenas, com isso, centenas de requests ao bucket para verificar se existe ou não.
Eu ainda acho melhor simplesmente pular essa verificação, mas se vc achar melhor, posso colocar.

Reply:
Sigo a relatoria. Boa!